python/ray/train/constants.py

```diff
@@ -68,11 +68,8 @@
 # as Trainable)
 DISABLE_LAZY_CHECKPOINTING_ENV = "TRAIN_DISABLE_LAZY_CHECKPOINTING"
 
-# Default NCCL_SOCKET_IFNAME.
-# Use ethernet when possible.
-# NCCL_SOCKET_IFNAME does a prefix match so "ens3" or "ens5" will match with
-# "en".
-DEFAULT_NCCL_SOCKET_IFNAME = "en,eth,bond"
+# Blacklist virtualized networking.
+DEFAULT_NCCL_SOCKET_IFNAME = "^lo,docker,vethc"
```

Comment:
oh actually the reason is that it should be `veth` not `vethc`. `c` happened to be the first hex character of the ID I sent you.

that also explains why it was present in only 12 or the 16 -- the other 4 must have had `c` as the first character of the id.

Reply:
Ah good catch...then now I'm wondering why this passed. Maybe this is not needed at all 🤔. Let me run the test with this default removed.